tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -246,15 +245,15 @@ Bridge network - ESX none
     Cleanup VIC Appliance On Test Server
 
 Bridge network - invalid
-    Pass execution  asdf
+    Run Keyword If  '%{HOST_TYPE}' != 'ESXi'  Pass Execution  Test skipped on vCenter
```

Comment:
**Super nitpick here**... would this: 
`Run Keyword If  '%{HOST_TYPE}' != 'ESXi'  Pass Execution  Test skipped on vCenter`
be the same as:
`Run Keyword If  '%{HOST_TYPE}' == 'VC'  Pass Execution  Test skipped on VC`
If it is, for consistency throughout this file, I would change it to match what we have already.